README.md

```diff
@@ -452,6 +453,24 @@ references:
 
 ![](img/manual-reference.png)
 
+#### Automatic Cross-References
+
+To refer to a header by its ID and have the cross-reference text automatically extracted,
+use the following syntax:
+
+`[](#header-id)`
```

Comment:
you may omit the link text like this: `[](#header-id)`

Reply:
Done